docs/system-tools/web-ui.md

```diff
@@ -4,4 +4,176 @@ title: Temporal Web UI
+| TEMPORAL_GRPC_ENDPOINT           | String representing server gRPC endpoint                                        | 127.0.0.1:7233                |
+| TEMPORAL_WEB_PORT                | HTTP port to serve on                                                           | 8088                          |
+| TEMPORAL_CONFIG_PATH             | Path to config file, see [configurations](#configuring-authentication-optional) | ./server/config.yml           |
+| TEMPORAL_PERMIT_WRITE_API        | Boolean to permit write API methods such as Terminating Workflows               | true                          |
+| TEMPORAL_WEB_ROOT_PATH           | The root path to serve the app under                                            | /                             |
+| TEMPORAL_HOT_RELOAD_PORT         | HTTP port used by hot reloading in development                                  | 8081                          |
+| TEMPORAL_HOT_RELOAD_TEST_PORT    | HTTP port used by hot reloading in tests                                        | 8082                          |
+| TEMPORAL_SESSION_SECRET          | Secret used to hash the session with HMAC                                       | "ensure secret in production" |
+| TEMPORAL_EXTERNAL_SCRIPTS        | Additional JavaScript tags to serve in the UI                                   |                               |
+| TEMPORAL_GRPC_MAX_MESSAGE_LENGTH | gRPC max message length (bytes)                                                 | 4194304 (4mb)                 |
+
+<details>
+<summary>
+Optional TLS configuration variables:
+</summary>
+
+| Variable                              | Description                                                         | Default |
+| ------------------------------------- | ------------------------------------------------------------------- | ------- |
+| TEMPORAL_TLS_CERT_PATH                | Certificate for the server to validate the client (web) identity    |         |
+| TEMPORAL_TLS_KEY_PATH                 | Private key for secure communication with the server                |         |
+| TEMPORAL_TLS_CA_PATH                  | Certificate authority (CA) certificate for the validation of server |         |
+| TEMPORAL_TLS_ENABLE_HOST_VERIFICATION | Enables verification of the server certificate                      | true    |
+| TEMPORAL_TLS_SERVER_NAME              | Target server that is used for TLS host verification                |         |
+
+- To enable mutual TLS, you need to specify `TEMPORAL_TLS_KEY_PATH` and `TEMPORAL_TLS_CERT_PATH`.
+- For server-side TLS you need to specify only `TEMPORAL_TLS_CA_PATH`.
+
+By default we will also verify your server `hostname`, matching it to `TEMPORAL_TLS_SERVER_NAME`. You can turn this off by setting `TEMPORAL_TLS_ENABLE_HOST_VERIFICATION` to `false`.
+
+</details>
+
+### Configuring Authentication (optional)
```

Comment:
```suggestion
### Configuring authentication
```
This causes the url to contain "-optional".
And generally I don't think it is a good practice.
I think it is easy enough to identify this section as optional in one of the first sentences.